packages/cdk-routing-split/test/steps/given.ts

```diff
@@ -0,0 +1,11 @@
+import { Key } from '../../src/add/interface';
+import { Chance } from 'chance';
+
+export const valid_api_request_body = (): any => {
```

Comment:
1) This is specific to "add" and should be reflected in the name
2) Commonly I see in tests these methods be valid, and then they be changed later as needed. I think this could end up just being "get_add_request_body"
3) However:
In Yan's example that heavily inspired this method, he's just returning an object from a json file out of the data folder in cases like this. It would get rid of a lot of boilerplate that isn't needed. You wouldn't be able to use chance though.

His given method is more for universal functions like a_random_user(), an_authenticated_user(), and get_appsync_context. It as me wondering, if wanted to use chance, if it would be worth having a separate "data" module with functions like this separate from the given? 
@nthornton2010  curious what you think too - this project is pretty small so may be overengineering

Reply:
@BennuAshes it does feel like a bit of over-engineering. I agree that there's some boilerplate, but its so small that it doesn't make much of a difference, especially after you've made one, it's just copy/paste at that point.